doc/deployment.rst

```diff
@@ -215,13 +215,23 @@ Start the ``fedmsg-hub`` daemon, which will pick up the datanommer plugin,
 which will in turn read in that connection string, start listening for
 messages, and store them all in the db.
 
+
 ::
 
     $ sudo systemctl start fedmsg-hub
     $ sudo systemctl enable fedmsg-hub
 
 You can check ``journalctl --follow`` for logs.
 
+Since fedmsg-hub stores the timestamp of the last message it had received 
```

Comment:
I think this would fit better in the [subscribing documentation in the consumer section](https://fedmsg.readthedocs.io/en/stable/subscribing/#the-consumer-approach). Add a sub-section called "Message Replay" or something and include this.

Additionally, the first sentence is a fragment sentence. Something like:

"fedmsg-hub stores the timestamp of the last message it has received in a file in the ``/var/run/fedmsg/status/fedmsg-hub/``. It uses this to replay messages it missed from the datagrepper service. To disable this functionality:"